documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1079,6 +1079,13 @@ ALTER DATABASE ADD SUPPLEMENTAL LOG DATA;
 The {prodname} Oracle connector requires that users accounts be set up with specific permissions so that the connector can capture change events.
 The following briefly describes these user configurations using a multi-tenant database model.
 
+[WARNING]
+====
+This user account should be used solely by the {prodname} Oracle connector to connect to the Oracle database and capture changes.
+This account should not be used to create tables or modify data as any change performed by this account will not be captured by the connector.
```

Comment:
Why is this the case actually?

Reply:
This was how the contribution originally came from Navis.  

Just a guess, but I suspect they were relying on the connector user to write records to the `HistoryRecorder` interface which they used to store all read LogMiner events in a database table for debuggin purposes.  It was likely easier to make this distinction in the LogMiner query than to add table exclusions I suppose.  Keep in mind this was all before we retrofitted the LogMiner query to apply table/schema include/exclude lists.

We can certainly look at relaxing this, but I think we should at a minimum keep `SYS` and `SYSTEM`.  wdyt?